### ISSUE TITLE
New Message - Manual input of receivers (Select Option Field)

### ISSUE THREAD

**Issue by SteveO1981** [nextcloud/mail#1863]
### Expected behavior
You manually enter a new e-mail address in the To field (in the Select option) and then you can send this recipient an e-mail.

### Actual behavior
A popup appears with a plus in front of where the new mail address is also displayed.
Then you can continue the input, but this is not completed in parallel in the popup field. A takeover of the complete e-mail address as a recipient becomes impossible. It only works occasionally after restarting the app several times.

### Mail app

**Mail app version:** 0.15.1

**Mailserver or service:** IMAP/SMTP (domainfactory)

### Server configuration

**Nextcloud Version:** 16.0.3

#### Client configuration
**Browser:** Firefox 64-bit

**Operating system:** Windows 10

**Comment by Valdnet** [comment]
It is not possible to select a given email address.
1. To:
![obraz](https://user-images.githubusercontent.com/47037905/61533802-2b1c4000-aa2e-11e9-81b2-c278237ff9cf.png)

2. CC:
![obraz](https://user-images.githubusercontent.com/47037905/61533861-51da7680-aa2e-11e9-80f4-e367cfd96edf.png)

3. BCC:
![obraz](https://user-images.githubusercontent.com/47037905/61533891-63bc1980-aa2e-11e9-9745-a57e8b23157d.png)

If you delete the e-mail addresses from the table `oc_mail_coll_addresses` manually from the database date, the function will work correctly until you send the first e-mail.

**Comment by ChristophWurst** [comment]
> [Valdnet](/Valdnet) pinned this issue [17 days ago](#event-2495895331)

could you please elaborate on what you're doing here?

**Comment by Valdnet** [comment]
@ChristophWurst I wanted to highlight the issue so that you would pay attention to this. In my opinion, the error is serious because you can not send email messages.

**Comment by ChristophWurst** [comment]
Please don't do this and leave pinning, labeling and any other meta information to the maintainers of the app.

**Comment by w1n5l0w** [comment]
I can confirm to experience this bug still in Mail app version 0.17.0 (Nextcloud-Server 16.0.5, Firefox 64-bit, Ubuntu Linux 18.04). Is there any progress in finding a cause or solution?

**Comment by ChristophWurst** [comment]
> Is there any progress in finding a cause or solution?

No

**Comment by laurentiu2** [comment]
Same here.
NC 16.0.7
Mail 0.17
Firefox 72.0.2 (Chrome as well)
Ubuntu 18.04
PHP 7.3

**Comment by jdaviescoates** [comment]
Same here, renders the Mail app completely unusable.

**Comment by Kugeleis** [comment]
same here
to field can't take more than on address. The weired drop down field confuses. It used to work in earlier versions flawlessly.

**Comment by alexanderdd** [comment]
https://github.com/nextcloud/mail/issues/2769 seems to be a duplicate of this issue but there is more discussion over there. 

Close this?

**Comment by ChristophWurst** [comment]
Yes. Thanks @alexanderdd.
